### PR TITLE
makepkg: add clean option

### PR DESCRIPTION
**Pull request by Managor**

### CODE REVIEW

File: pages/linux/makepkg.md [hunk 1]
```diff
@@ -19,3 +19,7 @@
 - Make a package, but skip source checksums:
 
 `makepkg --skipchecksums`
+
+- Clean the build folder before compiling:
```
Comment (navarroaxel) [inline pages/linux/makepkg.md:23]: The `--cleanbuild` arg cleans the `src/` folder before build, the `--clean` arg removes the `src/` and `pkg/` folders after build.
```suggestion
- Clean up work directories after a successful build:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Please update the link to `https://man.archlinux.org/man/makepkg.8`.

**Review by navarroaxel — APPROVED** [review]

**Review by SethFalco — APPROVED** [review]

File: pages/linux/makepkg.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 > Creates a package installable with the `pacman` package manager.
 > Runs the commands from a `PKGBUILD` file to build the package.
-> More information: <https://wiki.archlinux.org/index.php/Makepkg>.
+> More information: <https://man.archlinux.org/man/makepkg.8>.
```
Comment (CleanMachine1) [inline pages/linux/makepkg.md:5]:
```suggestion
> More information: <https://man.archlinux.org/man/makepkg.8>.
```

It redirects however you can remove the `.8` from the end.

**Review by CleanMachine1 — APPROVED** [review]

File: pages/linux/makepkg.md (hunk 2 above)
Comment (navarroaxel) [reply pages/linux/makepkg.md:5]: Check `grep -Rn 'man\.archlinux'` we always use the chapter suffix for Arch Linux's man pages.

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**